### PR TITLE
Create GeneticVariant_Alt_ID_Database_properties.mcf

### PR DESCRIPTION
**Pull request by spiekos**
Schema for properties of databases with alternative IDs for genetic variants generated by format_dbSNP_alt_ID_database_property_schema.py.

### CODE REVIEW

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf [hunk 1]
```diff
@@ -0,0 +1,1401 @@
+# Generated by script format_dbSNP_alt_ID_database_property_schema.py
+
+Node: dcid:fanconi_Anaemia_Mutation_Database_(FANCM)_ID
```
Comment (pradh) [inline schema/GeneticVariant_Alt_ID_Database_properties.mcf:3]: could we remove '(' and ')' from property names?

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf [hunk 2]
```diff
@@ -0,0 +1,1401 @@
...
+description: "The ID used by database tIDEX_University_of_British_Columbia for a genetic variant."
+
+Node: dcid:shworak_lab_George_Washington_University_ID
+name: "shworak_lab_George_Washington_University_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database shworak_lab_George_Washington_University for a genetic variant."
+
+Node: dcid:lDLR-LOVD_British_Heart_Foundation_ID
```
Comment (pradh) [inline schema/GeneticVariant_Alt_ID_Database_properties.mcf:24]: ldlr_LOVD or lDLR_LOVD? To be consistent about the use of "_"

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf [hunk 3]
```diff
@@ -0,0 +1,1401 @@
+# Generated by script format_dbSNP_alt_ID_database_property_schema.py
+
+Node: dcid:fanconi_Anaemia_Mutation_Database_(FANCM)_ID
+name: "fanconi_Anaemia_Mutation_Database_(FANCM)_ID"
```
Comment (pradh) [inline schema/GeneticVariant_Alt_ID_Database_properties.mcf:4]: Can we make these property names camelCase ?

We use '_' in property or class names very sparringly.

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf [hunk 4]
```diff
@@ -0,0 +1,1401 @@
...
+description: "The ID used by database genomic_Medicine_Lab_University_of_California_San_Francisco for a genetic variant."
+
+Node: dcid:klein_lab_Ludwig-Maximilians-University_ID
+name: "klein_lab_Ludwig-Maximilians-University_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database klein_lab_Ludwig-Maximilians-University for a genetic variant."
+
+Node: dcid:foundation_for_Research_in_Genetics_and_Endocrinology_FRIGE's_Institute_of_Human_Genetics_ID
```
Comment (pradh) [inline schema/GeneticVariant_Alt_ID_Database_properties.mcf:101]: Could we remove apostrophe?

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf [hunk 5]
```diff
@@ -0,0 +1,1401 @@
...
+description: "The ID used by database dhiti_Omics_Technologies_Pvt._Ltd. for a genetic variant."
+
+Node: dcid:liping_Wei_Laboratory_Peking_University_ID
+name: "liping_Wei_Laboratory_Peking_University_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database liping_Wei_Laboratory_Peking_University for a genetic variant."
+
+Node: dcid:australian_Inherited_Retinal_Disease_Registry_&_DNA_Bank_Sir_Charles_Gairdner_Hospital_ID
```
Comment (pradh) [inline schema/GeneticVariant_Alt_ID_Database_properties.mcf:1088]: Remove '&' from property names?

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf [hunk 6]
```diff
@@ -0,0 +1,1401 @@
...
+description: "The ID used by database neurogenetic_Laboratory_Oslo_University_Hospital for a genetic variant."
+
+Node: dcid:athena_Diagnostics_Inc_ID
+name: "athena_Diagnostics_Inc_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database athena_Diagnostics_Inc for a genetic variant."
+
+Node: dcid:dST/NWU_Preclinical_Drug_Development_Platform_North-West_University_ID
```
Comment (pradh) [inline schema/GeneticVariant_Alt_ID_Database_properties.mcf:1109]: Remove '/' from property names?

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf (hunk 1 above)
Comment (spiekos) [reply schema/GeneticVariant_Alt_ID_Database_properties.mcf]: removed

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf (hunk 2 above)
Comment (spiekos) [reply schema/GeneticVariant_Alt_ID_Database_properties.mcf]: changed to camel case

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf (hunk 3 above)
Comment (spiekos) [reply schema/GeneticVariant_Alt_ID_Database_properties.mcf]: changed to camel case

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf (hunk 5 above)
Comment (spiekos) [reply schema/GeneticVariant_Alt_ID_Database_properties.mcf]: changed & to And

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf (hunk 6 above)
Comment (spiekos) [reply schema/GeneticVariant_Alt_ID_Database_properties.mcf]: removed / and other problematic symbols from names

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf [hunk 7]
```diff
@@ -0,0 +1,1023 @@
+# Generated by script format_dbSNP_alt_ID_database_property_schema		.py
```
Comment (tjann) [inline schema/GeneticVariant_Alt_ID_Database_properties.mcf:1]: not a big deal, but there's an interesting whitespace here

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf [hunk 8]
```diff
@@ -0,0 +1,1023 @@
...
+description: "The ID used by database Donald Williams Parsons Laboratory Baylor College Medicine for a genetic variant."
+
+Node: dcid:geneticsInfertilityAndPreimplantationGeneticDiagnosisCentreHospitalierUniversitaireGrenobleAlpesID
+name: "geneticsInfertilityAndPreimplantationGeneticDiagnosisCentreHospitalierUniversitaireGrenobleAlpesID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database Genetics Infertility And Preimplantation Genetic Diagnosis Centre Hospitalier Universitaire Grenoble Alpes for a genetic variant."
+
+Node: dcid:departmentPathologyAndMolecularMedicineQueen'sUniversityID
```
Comment (tjann) [inline schema/GeneticVariant_Alt_ID_Database_properties.mcf:115]: let's remove the ' to be safe

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf (hunk 4 above)
Comment (tjann) [reply schema/GeneticVariant_Alt_ID_Database_properties.mcf]: I see shanth had the same comment, so @spiekos  could you remove the ' ?

**Review by tjann — CHANGES_REQUESTED** [review]

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf (hunk 4 above)
Comment (spiekos) [reply schema/GeneticVariant_Alt_ID_Database_properties.mcf]: done

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf (hunk 7 above)
Comment (spiekos) [reply schema/GeneticVariant_Alt_ID_Database_properties.mcf]: removed

File: schema/GeneticVariant_Alt_ID_Database_properties.mcf (hunk 8 above)
Comment (spiekos) [reply schema/GeneticVariant_Alt_ID_Database_properties.mcf]: done

**Review by pradh — APPROVED** [review]
This is looking good. But before merging please resolve conflict on GenVarSource_enums.mcf.

**Review by tjann — APPROVED** [review]

**Pull request merged**